functions/routes/index.js

```diff
@@ -24,6 +22,7 @@ router.post("/task", async function(req, res, next) {
             res.status(500).send("Area not serviced");
             return;
         }
+        onfleetTeamId = teamData.OnFleetID;
```

Comment:
The reason we have two of them is to support the two conditions. If there isn't a team it will create a new team and set the onfleetteamId to that. If there is a team then it just grabs the id from the teamData